README.md

```diff
@@ -36,7 +36,8 @@ The charm may provide defaults for these service configuration
   the | character indicates that the value is a multiline string):
 
   ```yaml
-  install_sources: |
+  install_sources:
+      default: |
       - ppa:stub/cassandra
```

Comment:
I'd probably prefer to see the indentation for the following lines increased as well, just for aesthetics, but I don't think it would cause a technical issue.